.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+name: Release
+
+on: [workflow_dispatch]
+
+jobs:
+  build:
+    name: Build the project
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Setup JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: 11
+          java-package: jdk
+          distribution: adopt
+          architecture: x64
+          # this also creates settings.xml with the following server
+          server-id: sonatype-nexus-staging # Value of the distributionManagement/repository/id field of the pom.xml
+          server-username: SONATYPE_USERNAME # env variable for username in deploy
+          server-password: SONATYPE_PASSWORD # env variable for token in deploy
+          # only signed artifacts will be released to maven central. this sets up things for the maven-gpg-plugin
+          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }} # Value of the GPG private key to import
+          gpg-passphrase: GPG_PASSPHRASE # env variable for GPG private key passphrase
+          settings-path: ${{ github.workspace }}
+      - name: Cache local Maven repository
+        uses: actions/cache@v2.1.6
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Release artifacts
+        run: mvn -Dresume=false release:prepare release:perform -Djib.skip -DskipTests -Darguments="-s $GITHUB_WORKSPACE/settings.xml"
```

Comment:
set release version and development version here?

Reply:
done